storage/src/bmt/mod.rs

```diff
@@ -41,13 +41,17 @@
 //! ```
 
 use bytes::{Buf, BufMut};
-use commonware_codec::{EncodeSize, Read, ReadRangeExt, Write};
-use commonware_cryptography::Hasher;
+use commonware_codec::{EncodeSize, Read, ReadExt, ReadRangeExt, Write};
+use commonware_cryptography::{Digest, Hasher};
 use thiserror::Error;
 
-/// There should never be more than 255 siblings in a proof (would mean the Binary Merkle Tree
+/// There should never be more than 255 levels in a proof (would mean the Binary Merkle Tree
 /// has more than 2^255 leaves).
-const MAX_SIBLINGS: usize = u8::MAX as usize;
+///
+/// Note: This constant represents the maximum tree height, NOT the maximum number of siblings
```

Comment:
seems a weird thing to clarify? (I would have never assumed something called MAX_LEVELS would be a max on siblings.)

Reply:
This was a remnant of an earlier revision, will change